src/app/shared/services/login/login.service.ts

```diff
@@ -199,6 +203,15 @@ export class LoginService {
     }
   }
 
+  public wrongNetwork() {
+    if (this.isSessionActive()) {
+      this.openSwal(
+        { title: 'Ops!', text: "You're using wrong network" },
```

Comment:
"Oops!" (extra o). "You are using the wrong network" (add 'the')

Reply:
Should we maybe specify "MetaMask network"?

Reply:
I can check for adding a button to import/switch to correct metamask configuration. But don't know if SWAL lib allows me to do it.

Reply:
I was able to add it. However I will improve it by creating AlertService (wrapper for swal). This way will be easier to test functions that uses Swal. To be honest, i do not know how to mock swal 😅